modules/objdetect/misc/python/test/test_qrcode_detect.py

```diff
@@ -4,15 +4,15 @@
 QR code detect and decode pipeline.
 ===============================================================================
 '''
-
+import os
 import numpy as np
 import cv2 as cv
 
 from tests_common import NewOpenCVTests
 
 class qrcode_detector_test(NewOpenCVTests):
     def test_detect_and_decode(self):
-        img = cv.imread(self.extraTestDataPath + '/cv/qrcode/link_ocv.jpg')
+        img = cv.imread(os.path.join(self.extraTestDataPath, '/cv/qrcode/link_ocv.jpg'))
```

Comment:
I would suggest removing slash too (there is no absolute path):

```.diff
'/cv/
'cv/
```

Also it would be nice to check that input data is read properly (`img != None`)